website/docs/tutorials/08-interacting-with-starknet/scripting.md

```diff
@@ -0,0 +1,194 @@
+# Scripting
+In order to automate a process of your desire that includes protostar operations, you may want to assemble some protostar commands and put them in a single script.
+
+This tutorial shows a simple example of how to do such a thing using scripting in bash.
+
+### File Structure
+
+Let's create the following file structure:
```

Comment:
```suggestion
Let's assume the following file structure:
```
Maybe would be better to just you standard template created by protostar init? User may want to do it alongside the tutorial

Reply:
Great idea of using the `protostar init`, haven't thought of that, thanks!